Sources/Map/OATerrainLayer.mm

```diff
@@ -76,7 +77,8 @@ - (instancetype) init:(EOATerrainLayerType)terrainType
 - (void) onTerrainResourcesChanged
 {
     [self initCollection];
-    [[OsmAndApp instance].data.terrainChangeObservable notifyEvent];
+    [[NSNotificationCenter defaultCenter] postNotificationName:kNotificationSetProfileSetting
```

Comment:
The kNotificationSetProfileSetting should be posted by OAAppSettings only (as it is done now)!!!